app/src/main/res/layout/fragment_browser.xml

```diff
@@ -47,9 +47,6 @@
         android:id="@+id/cursorView"
         android:layout_width="@dimen/remote_cursor_size"
         android:layout_height="@dimen/remote_cursor_size"
-        android:layout_gravity="top|start"
```

Comment:
nit: For defensive coding, it might be helpful to leave this (e.g. if Android decides to layout us somewhere other than the origin and all of our draw calls are off).

Reply:
We programmatically set our starting position when the view is first measured, so this shouldn't matter.  I can add a comment saying that though.